pkg/container/run.go

```diff
@@ -25,17 +25,17 @@ import (
 	"sigs.k8s.io/kind/pkg/exec"
 )
 
-// Docker container IDs are hex, more than one character, and on their own line
+// Container IDs are hex, more than one character, and on their own line
 var containerIDRegex = regexp.MustCompile("^[a-f0-9]+$")
 
-// Run creates a container with "docker run", with some error handling
+// Run creates a container with "ContainerEngine run", with some error handling
```

Comment:
let's just say Engine since at least that maps to the var we're using. the reason for mentioning "docker run" is to correlate with the documented behavior there without documenting it ourselves.